<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LootTableEditorDock</class>
 <widget class="QDockWidget" name="LootTableEditorDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loot table editor</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="luckBasedCheck">
      <property name="text">
       <string>Luck based</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="lootTableTypeCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Empty</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Entity drops</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Block drops</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Treasure chest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fishing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cat or villager gift</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Advancement reward</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Generic (none of the above)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QPushButton" name="writeLootTableBtn">
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="readLootTableBtn">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="2">
       <widget class="QToolButton" name="deletePoolButton">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QListView" name="poolListView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>200</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="addPoolButton">
        <property name="toolTip">
         <string>Add pool</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QTabWidget" name="poolEditor">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="rollsTab">
       <attribute name="title">
        <string>Rolls</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="toolTip">
           <string>The number of times the loot table will pick an item from its pool.</string>
          </property>
          <property name="text">
           <string>Rolls:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="NumericInput" name="rollsInput">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="bonusRollsLabel">
          <property name="text">
           <string>Bonus rolls:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="NumericInput" name="bonusRollsInput">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="conditionsTab">
       <attribute name="title">
        <string>Consitions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item row="1" column="0">
         <widget class="QPushButton" name="addCondButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QScrollArea" name="conditionsArea">
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="conditionsContainer">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>474</width>
             <height>279</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="functionsTab">
       <attribute name="title">
        <string>Functions</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="entriesTab">
       <attribute name="title">
        <string>Entries</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="addEntryButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QScrollArea" name="entriesArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="entriesContainer">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>474</width>
             <height>274</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumericInput</class>
   <extends>QFrame</extends>
   <header location="global">NumericInput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
